### ISSUE TITLE
populates option in firestoreConnect error - [object Array] is not a function

### ISSUE THREAD

**Issue by AgnesToulet** [prescottprue/redux-firestore#250]
Hello, I'm trying to work with the populates options of redux-firestore and I'm facing an error.

**What is the current behavior?**
I'm using the populates option as it's described in the documentation, here's an extract of my React component calling firestoreConnect:
```export default compose(
  withStyles(styles),
  firestoreConnect(props => {
    return [{
      collection: 'recipes',
      doc: props.match.params.recipeId,
      subcollections: [{ collection: 'ingredients' }],
      populates: [{ child: 'foodID', root: 'foods' }],
      storeAs: 'recipeFoods',
    }]
  }),
  connect(mapStateToProps),
)(RecipeDetails);
```
And I have an error in my console:
![image](https://user-images.githubusercontent.com/35176601/69336631-3e028380-0c5f-11ea-9c9c-0ed5515e1c81.png)

**What is the expected behavior?**
It should work without error.

**Which version of redux-firestore are you using? What about other dependencies?**
I'm using version ^0.10.0 of redux-firestore and ^3.0.5 of react-redux-firebase

**Which environments/browsers are affected by this issue? Did this work in previous versions or setups?**
I'm on Chrome. I just started this project so I've never seen it working.
I looked through the file in question (in my node_modules > es > utils > query.js) and updated the function promisesForPopulate by replacing 
```populatesForData.some(populatesForData, function (populate) {```
with
```populatesForData.some(function (populate) {```
and the error disappeared.
I get the referenced objects in my firestore data but no "populated" field.

**Minimal demo to reproduce issue (using [codesandbox](https://codesandbox.io/) or similar)**
I don't have one but I faced this issue as soon as I added the populates option in my query. Even if the query is very simple, the same error occurs.

**Comment by prescottprue** [comment]
@AgnesToulet thanks for reaching out! It seems like that may be from a recent update replacing lodash functions with built in JS methods.

**Comment by yumekun** [comment]
i meet issue when i use queryoption at orderByoption[] , 
{
  collection: 'cities',
  orderBy: [
    ['name'],
    ['email
![issue](https://user-images.githubusercontent.com/55476094/69425815-3aa1f180-0d5e-11ea-8953-549ec2cd2cd2.png)
', 'desc']
  ]
}

this the error message
Argument of type '({ collection: string; where: [string, "==", boolean]; orderBy: [string[], string[]]; limit: number; } | { collection: string; })[]' is not assignable to parameter of type 'string | ReduxFirestoreQuerySetting | ReduxFirestoreQuerySetting[] | string[] | mapper<{}, (string | ReduxFirestoreQuerySetting)[]> | mapper<{}, string | ReduxFirestoreQuerySetting> | undefined'.
  Type '({ collection: string; where: [string, "==", boolean]; orderBy: [string[], string[]]; limit: number; } | { collection: string; })[]' is not assignable to type 'ReduxFirestoreQuerySetting[]'.
    Type '{ collection: string; where: [string, "==", boolean]; orderBy: [string[], string[]]; limit: number; } | { collection: string; }' is not assignable to type 'ReduxFirestoreQuerySetting'.
      Type '{ collection: string; where: [string, "==", boolean]; orderBy: [string[], string[]]; limit: number; }' is not assignable to type 'ReduxFirestoreQuerySetting'.
        Types of property 'orderBy' are incompatible.
          Type '[string[], string[]]' is not assignable to type '[string, OrderByDirection] | [string, OrderByDirection][] | undefined'.
            Type '[string[], string[]]' is not assignable to type '[string, OrderByDirection]'.
              Type 'string[]' is not assignable to type 'string'.

**Comment by prescottprue** [comment]
Fixed in [v0.11.0 release](https://github.com/prescottprue/redux-firestore/releases/tag/v0.11.0). Thanks for posting!
